### PR TITLE
Update shared base and add content components

### PR DESCRIPTION
**Pull request by YuCJ**
- [x] Add `Brief`
- [x] Add `Infobox`
- [x] Add `CenteredQuote`
- [x] Add `Paragraph`
- [x] Add `Blockquote`
- [x] Add `Annotation`

### CODE REVIEW

**Comment by YuCJ** [comment]
Add a commit to upgrade `lodash` due to vulnerabilities alert

File: dev/entry.js [hunk 1]
```diff
@@ -3,4 +3,32 @@ import mockPost from './mock-post.json'
...
+      infoboxTitle: '#494949',
+      line: '#70f197',
+      link: '#55e07f',
+      metadataDefault: '#808080',
+      metadataPrimary: '#55e07f',
+      metadataSecodary: '#55e07f',
+      paragraph: '#494949',
+      quote: '#838383',
+      sectionSubtitle: '#494949',
+      sectionTitle: '#494949',
```
Comment (nickhsine) [inline dev/entry.js]: Do we need to specify all the colors we are going to use in `ThemeProvider`?

In mockup, the default colors(like grey, black) are almost the same.

Could we only provide primary and secondary color?
```
<ThemeProvider theme={{
  color: {
    primary: {
      text: '#55e07f',
      line: '#70f197',
      shape: '#73fa9c',
    },  // primary set for green theme
    secondary: '#7e33f6', // purple
  },
}}
>
```

Comment (nickhsine) [reply dev/entry.js]: Could we let each component to decide the colors by themselves?

Comment (YuCJ) [reply dev/entry.js]: Updated

**Comment by nickhsine** [comment]
LGTM 👍

**Pull request merged**